ee/main.go

```diff
@@ -153,6 +153,7 @@ func main() {
 		PrimeHubCache: primehubCache,
 		PhfsEnabled:   phfsEnabled,
 		PhfsPVC:       phfsPVC,
+		PrimehubUrl:   viper.GetString("application.primehubUrl"),
```

Comment:
Is it possible to consolidate this config with `modelDeployment.primehubUrl`?

Make it to the toplevel just like graphqlSecret https://github.com/InfuseAI/primehub-controller/blob/master/config.yaml.example#L2

And make sure we update the config.yaml.example file if new config added.